templates/master.yaml

```diff
@@ -496,7 +395,7 @@ coreos:
       Type=oneshot
       RemainAfterExit=yes
       TimeoutStartSec=0
-      Environment="IMAGE={{.Cluster.Operator.NetworkSetup.Docker.Image}}
+      Environment="IMAGE={{.Spec.Docker.ImageNamespace}}/setup-network-env:{{.Spec.Operator.NetworkSetupVersion}}"
```

Comment:
It should be `.Cluster.Operator.NetworkSetup.Docker.Image`. See https://github.com/giantswarm/clustertpr/blob/47bae764f0a168b9dac27e274515b1a5d03eea6b/operator/networksetup/docker/docker.go#L4.